docs/sources/flow/tasks/send-profiles-to-pyroscope/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/send-profiles-to-pyroscope/
+canonical: https://grafana.com/docs/agent/latest/flow/send-profiles-to-pyroscope/
+description: Send profiles to Pyroscope 
+title: Send profiles Pyroscope 
+weight: 120
+---
+
+# Send profiles to Pyroscope
+
+Learn how to configure {{< param "PRODUCT_NAME" >}} to collect [Pyroscope Server][] profiles and forward them to a [Pyroscope Server][].
+
+This topic describes how to:
+
+* Setup [Pyroscope Server][].
+* Setup {{< param "PRODUCT_NAME" >}} to collect profiles and send them to [Pyroscope Server][].
+
+## Components used in this topic
+
+* [pyroscope.write][]
+* [pyroscope.scrape][]
+
+## Before you begin
+
+* Download {{< param "PRODUCT_NAME" >}}.
+* Install [docker][] if it is not already installed.
+
+## Steps
```

Comment:
This looks like a tutorial to me. See https://github.com/grafana/agent/blob/main/docs/developer/writing-docs.md

A tutorial:
> Provides procedures that users can safely reproduce and learn from. Answers the question: “Can you teach me to …?”

While task:
> Tasks are production-ready and contain best practices and recommendations. They are quite detailed, with Reference pages being the only type of documentation that has more detail.

I believe we want a task doc, which will help users configure sending to Pyroscope in production, instead of on a local machine. I know that config is roughly the same, but there's a distinction in our docs now between tasks and tutorials that I'd like to maintain.

cc @clayton-cornell to chime in, if we need a second opinion ;)

Reply:
@thampiotr @mattdurham Yes, I was thinking the same thing. That's one of the reasons I want to re-review this PR once some of the questions I raised are cleared up. 

I agree, it's more tutorial than task. For the tasks we need to define what the purpose or goal is, and it should generally be a single thing... a single goal such as (only) configure an existing setup to send Pyroscope in production. We make quite a few assumptions such as... Agent is already installed... and we focus on one thing... the configuration needs.